specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/outputs.json

```diff
@@ -490,6 +490,31 @@
         "type"
       ]
     },
+    "RawOutputDatasource": {
+      "description": "Describes a raw output data source.",
```

Comment:
NIT: The comment content you had for inputs also applies to outputs (i.e. can't create a streaming job with raw output, required on create in descriptions where appropriate, etc.